packages/amplication-server/src/core/serviceMessageBrokerConnection/dto/ServiceMessageBrokerConnectionUpdateInput.ts

```diff
@@ -0,0 +1,17 @@
+import { Field, InputType } from '@nestjs/graphql';
+import { BlockUpdateInput } from '../../block/dto/BlockUpdateInput';
+
+@InputType({
+  isAbstract: true
+})
+export class ServiceMessageBrokerConnectionUpdateInput extends BlockUpdateInput {
+  @Field(() => String, {
+    nullable: false
+  })
+  messageBrokerId!: string;
+
+  @Field(() => Boolean, {
+    nullable: false
+  })
+  enabled!: boolean;
+}
```

Comment:
Why not give the option for a partial update?

Reply:
Because these properties are saved in a JSON object in the data, and I don't think we are handling the merging of existing and new values correctly in this field. 
Since we control the API and the client, the current design will allow us to have less errors and move faster.

Reply:
@tupe12334 
This is a good comment, can you add something about it here?
https://www.notion.so/amplication/Creating-a-new-Block-Type-1d70ff31ae444aceba412fc0f45b28a0